### ISSUE TITLE
Jamf Pro Configuration Profile Json

### ISSUE THREAD

**Issue by lukecater** [Jamf-Concepts/Setup-Manager#51]
I might be overlooking something, but I can't find the "wait" action in the Enrolment Action array for either version 1.0 or 1.1.

<img width="1336" alt="Screenshot 2024-10-24 at 11 21 12 AM" src="https://github.com/user-attachments/assets/acd50dbc-8bd2-4573-8ef6-b79c01ff094a">

**Comment by scriptingosx** [comment]
Yes, unfortunately there are technical limitations in the custom schema JSON which mean that we cannot provide the complete functionality in the custom JSON. (Localization is another that is not available.)

You will have to go to a custom plist if you want to use a `wait` action.

However, if you want to use a `wait` action because you are experiencing issues with Jamf Pro, please test the 1.1beta and let us know if that resolves the issue without needing to add a `wait` action.

**Comment by scriptingosx** [comment]
I will take this as documentation request. This should be added.

**Comment by lukecater** [comment]
Thank you! I've been using the beta, and it's performing really well during the testing phase—kudos! In my workflow, after changing a computer's name, It download a few more profiles and Mac apps based on the scope, so I thought a brief wait might be beneficial. I really appreciate having the JSON as a foundational element; I can easily adjust it by modifying the plist.

**Comment by scriptingosx** [comment]
fixed in the 1.1 documentation
